src/components/hero/DemoButton.tsx

```diff
@@ -25,6 +30,15 @@ interface Props {
 
 function DemoButton({ step, type }: Props) {
     const navigate = useNavigate();
+
+    const adminCapabilities = useEntitiesStore_capabilities_adminable();
+    const objectRoles = Object.keys(adminCapabilities);
+
+    const readCapabilities = useEntitiesStore_capabilities_readable();
+    const demoAccessExists = Object.keys(readCapabilities).includes('demo/');
+
+    const [open, setOpen] = useState(false);
+
     const goToFilteredTable = useCallback(() => {
```

Comment:
We need to eventually move this function into a shared hook so we don't have to pass it down.